pkgs/development/compilers/xa/default.nix

```diff
@@ -0,0 +1,47 @@
+{ stdenv
+, lib
+, fetchzip
+}:
+
+stdenv.mkDerivation rec {
+  pname = "xa";
+  version = "2.3.11";
+
+  src = fetchzip {
+    # Old versions are moved to dists/unsupported subdirectory
```

Comment:
While rebasing I noticed that the package was already submitted by someone else since I started working on this. I've dropped my version and added `urls` to the affected packages instead.